doc/layout-config-reference.rst

```diff
@@ -14,6 +14,10 @@ API Version
 MMC Options
 -----------
 
+``disklabel`` (string)
+   Partitioning style for the device. Available options are ``msdos`` (default)
+   and ``gpt``.
```

Comment:
"Partitioning style" is somewhat imprecise wording here. Better use "partition table". Be aware that GPT already contains this in its abbreviation, so something like:
```
MS-DOS partition table
MBR partition table
GUID Partition Table (GPT)
```
are valid terms.

Please also replace "Available options" with "Currently supported options", as there are more options which can be use in disklabel, available by libparted, but not supported by partup.